modules/imgcodecs/perf/perf_main.cpp

```diff
@@ -8,3 +8,35 @@
 #endif
 
 CV_PERF_TEST_MAIN(imgcodecs)
+
+namespace opencv_test
+{
+
+using namespace perf;
+
+#define TYPICAL_MAT_SIZES_TIFF_DECODE  cv::Size(128, 128), cv::Size(256, 256), cv::Size(512, 512), cv::Size(1024, 1024)
```

Comment:
I propose to add at least VGA (640x480), hd (1280x720) and fullhd (1920x1080) resolutions. They are popular in computer vision apps.